_episodes/02-file_parsing.md

```diff
@@ -300,35 +315,37 @@ for linenum, line in enumerate(list_name):
 
 In this notation, there are now *two* variables you can use in your loop commands, `linenum` (which can be named something else) will keep up with what iteration you are on in the loop, in this case what line you are on in the file. The variable `line` (which could be named something else) functions exactly as it did before, holding the actual information from the list.  Finally, instead of just giving the list name you use `enumerate(list_name)`.  
 
+**Important note**: `enumerate(list_name)` will start with 0-index so the first line will be label as '0', to change this behavior, use `start` variable in enumerate.
+
 This block of code searches our file for the line that contains "Center" and reports the line number.
 ```
-for linenum, line in enumerate(data):
+for linenum, line in enumerate(data, start=1):
```

Comment:
Cool trick! Didn't know about `start` in enumerate. If we include this it should be in a callout and not part of the main material.

Reply:
Can you please remove `start` here and add this capability as a callout as well?

Typically for call outs it is material that we may or may not teach in person depending on the level of the group.

Reply:
I have changed it back to the original together with other lines affected by this!